docs/faq/code-analysis/does-codacy-check-for-dependencies.md

```diff
@@ -1,5 +1,5 @@
 # Does Codacy check for dependencies?
 
-Codacy doesn't check for dependencies. Codacy performs static code analysis on the source code and checks for code style and security issues, duplication, complexity, and coverage.
+Yes, Codacy scans the manifest files of your repositories and displays any vulnerable dependencies as issues in your Codacy dashboard.
```

Comment:
The **Codacy dashboard** is the Security monitor, right? If it is, we should use the name of the product feature, and we can also add a link to its page. WDYT?

```suggestion
Yes, Codacy scans the manifest files of your repositories and displays any vulnerable dependencies as issues in the [Security monitor](../../repositories/security-monitor.md).
```

Reply:
Not necessarily only the Security monitor. Since these are generic Codacy Issues, they are displayed everywhere a normal issue is displayed: in the Issues tab, in the Security monitor if it's a security issue. In fact, now that you point this out, it may be simpler to call them Issues and not be too specific. Thoughts?

```suggestion
Yes, Codacy scans the manifest files of your repositories and displays any vulnerable dependencies as Codacy issues.
```

Reply:
Yes, makes sense!